README.md

```diff
@@ -32,6 +32,17 @@ If you want to find out all the magic behind, how to override the default values
 ### ArgoCD Master and Child 👩‍👦
 We can create a master ArgoCD instance in the cluster that can bootstrap other "child" ArgoCD instance(s) for any given project team. This is a good approach if you want each project team to own and operate their own software development tools (jenkins, sonar, argocd, etc) but restrict any elevated permissions they may need e.g.creating argocd Custom Resources Definitions (`CRD's`) or limiting project creation. See [ArgoCD Master and Child Deployment](docs/argocd-master-child.md)
 
+
+### Cleanup 
+Uninstall and delete all resources in the various projects
+```bash
+# delete ubiquitous-journey
+helm template -f argo-app-of-apps.yaml ubiquitous-journey/ | oc -n labs-ci-cd delete -f-
+# delete and clean-up bootstrap, run:
+helm template bootstrap --dependency-update -f bootstrap/values-bootstrap.yaml bootstrap | oc delete -f-
```

Comment:
Since the installation step is now as `helm install..`, this step should be as follow:

```
helm delete bootstrap --namespace labs-bootstrap 
```

Reply:
Fixed now